mico-admin/src/app/api/api.service.ts

```diff
@@ -178,7 +178,8 @@ export class ApiService {
         const resource = 'applications/';
 
         return this.rest.post<ApiObject>(resource, data).pipe(flatMap(val => {
-            this.getApplications();
+            this.getApplications().subscribe();
```

Comment:
getApplications has an internal subscribe to the http request. As such the request will be performed whether this subscribe here is present or not.